src/main/java/blusunrize/immersiveengineering/client/render/tile/BottlingMachineRenderer.java

```diff
@@ -102,8 +103,8 @@ else if(processTimer <= 65)
 					lift = (85-processTimer)/20f*.125f;
 					itemFill = 1;
 				}
-				itemY += lift;
-				lift += .0625;
+				if (lift > pixelHeight)
+				itemY += lift-pixelHeight;
```

Comment:
Is this supposed to be part of the if? should be indented as such.

Reply:
yep! fixed